docs/changelog_v1.0.md

```diff
@@ -0,0 +1,596 @@
+## Changelog
+
+### v1.0.0.rc0 (18/2/2022)
+
+#### Compatibility
+
+- We refactor our three coordinate systems to make their rotation directions and origins more consistent, and further remove unnecessary hacks in different datasets and models. Therefore, please re-generate data infos or convert the old version to the new one with our provided scripts. We will also provide updated checkpoints in the next version. Please refer to the compatibility documentation for more details.
```

Comment:
should the compatibility doc link added in Changelog?

Reply:
Good suggestion :)